pkg/adapter/adapter.go

```diff
@@ -306,9 +306,19 @@ func (a *Adapter) PollForMessages(channel *wabbit.Channel,
 			return nil
 		case msg, ok := <-msgs:
 			if !ok {
```

Comment:
is there a way to log here what was wrong? Or we have this logs from rabbitmq client?

Reply:
Yep, this is coming from the rabbitmq client, I'm logging the message cause it probably gives that extra context